src/libcore/slice/mod.rs

```diff
@@ -16,6 +16,9 @@
 
 #![stable(feature = "rust1", since = "1.0.0")]
 
+// FIXME: replace remaining ... by ..= after next stage0
+// Silence warning: "... is being replaced by ..="
+#![cfg_attr(not(stage0), allow(warnings))]
```

Comment:
Errr is there no way to silence just the `` `...` is being replaced by `..=` `` message, instead of all warnings?

Reply:
It's because the `warn_dotdoteq` function just shows a generic warning without any flag.